pkgs/applications/networking/instant-messengers/signal-desktop/update.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash nix-update curl coreutils
+
+set -ex
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+case "$UPDATE_NIX_ATTR_PATH" in
+  signal-desktop)
+    latestTag=$(curl https://api.github.com/repos/signalapp/Signal-Desktop/releases/latest | jq -r ".tag_name")
```

Comment:
All calls to api.github.com should optional take a GITHUB_TOKEN

Reply:
fixed